#include "ChunkServerServiceImpl.hpp"
int ChunkServerServiceImpl::SetChunkStatus(const ::chunkserver::SetChunkStatusReq & oReq, ::chunkserver::SetChunkStatusRsp & oResp) {
    return -1;
}
int ChunkServerServiceImpl::AllocateInode(const ::chunkserver::AllocateInodeReq & oReq, ::chunkserver::AllocateInodeRsp & oResp) {
    return -1;
}
int ChunkServerServiceImpl::ReadSlice(const ::chunkserver::ReadSliceReq & oReq, ::chunkserver::ReadSliceRsp & oResp) {
    return -1;
}
int ChunkServerServiceImpl::WriteSlice(const ::chunkserver::WriteSliceReq & oReq, ::chunkserver::WriteSliceRsp & oResp) {
    return -1;
}
